include/NovelRT.h

```diff
@@ -81,27 +81,39 @@
 //libpng
 #include <png.h>
 
+/**
+ * NovelRT is a cross-platform, flexible Visual Novel and 2D game engine.
+ * It is aimed at designers and developers alike, however many of the designer tools and features we have on our roadmap have yet to be implemented.
+ */
 namespace NovelRT {
   typedef class Atom Atom;
   typedef class DebugService DebugService;
   typedef class LoggingService LoggingService;
   typedef class NovelRunner NovelRunner;
 }
-
+/**
+ * Contains animation features.
+ */
 namespace NovelRT::Animation {
   typedef class SpriteAnimator SpriteAnimator;
 }
-
+/***
+ * Contains audio features, such as playing audio, and managing audio resources.
+ */
 namespace NovelRT::Audio {
   typedef std::vector<ALuint> SoundBank;
   typedef std::vector<ALuint> MusicBank;
   typedef class AudioService AudioService;
 }
-
+/**
+ * Contains features to interop with the .NET Core runtime.
+ */
 namespace NovelRT::DotNet {
   typedef class RuntimeService RuntimeService;
 }
-
+/**
+ * Contains graphics features, such as rendering, textures, cameras, etc...
```

Comment:
Nit: that can just be `etc.` since `etc...` reads a bit weirdly for documentation.

Reply:
Hm, I'm used to doing `etc...` but makes sense.